docs/documentation/actions/kafka.md

```diff
@@ -0,0 +1,130 @@
+!!! important "Target Configuration"
+    For all actions, a target should be defined and have a `name` and a valid `url`.
+
+=== "minimal config"
+    ```json title="Kafka target example"
+    {
+      "name": "my_kafka_target"
+      "url": "tcp://localhost:60962"
+    }
+    ```
+
+=== "extended config"
+    ```json title="Kafka target example"
+    {
+        "name":"KAFKA",
+        "url":"tcp://kafka.server.fr:9095",
+        "properties":[
+            {
+            "key":"ssl.keystore.location",
+            "value":"/keystores/keys.jks"
+            },
+            {
+            "key":"security.protocol",
+            "value":"SSL"
+            },
+            {
+            "key":"ssl.keystore.password",
+            "value":"password"
+            },
+            {
+            "key":"ssl.truststore.password",
+            "value":"password"
+            },
+            {
+            "key":"ssl.truststore.location",
+            "value":"/truststores/trust.jks"
+            },
+            {
+            "key":"auto.offset.reset",
+            "value":"earliest"
+            },
+            {
+            "key":"enable.auto.commit",
+            "value":"true"
+            }
+        ]
+    }
+    ```
+
+# Publish
+Use this action to publish a message to a Kafka topic.
+
+=== "Inputs"
+
+    | Required | Name         | Type                             |  Description                                             |
+    |:--------:|:-------------|:---------------------------------|:--------------------------------------------------------:|
+    |    *     | `target`     | {name: string, url: string}      | Kafka target                                             |
```

Comment:
target type is just a String of the target name

Reply:
Here we are talking about KafkaBasicPublishAction.java or ChutneyStepBuilder.KafkaBasicPublishTask in kotlin ?

Reply:
we should be consistent with other documentation page. 
I think we have to have the type of the pivot format => json.
For duration, we should document : Duration (String)